products/sle/main.pm

```diff
@@ -682,6 +682,9 @@ sub load_consoletests() {
         loadtest "console/zypper_ref";
         loadtest "console/yast2_lan" unless is_bridged_networking;
         loadtest "console/curl_https";
+        if (get_var('SCC_ADDONS', '') =~ /asmm/) {
```

Comment:
did you actually try it ? probably you not correctly understand how it works - 
for example string 'hpcm asmm extratext' will return true for `check_var_array('SCC_ADDONS', 'asmm')`